src/screens/ProfileScreen.tsx

```diff
@@ -71,7 +71,7 @@ export const ProfileScreen = ({ navigation }: IProfileScreenProps) => {
                   // Fetch token *before* signing out
                   const token = await user!.getIdToken();
                   await signOut();
-                  await userApi.deleteUserProfile({
+                  userApi.deleteUserProfile({
```

Comment:
Dont you need await for API calls?

Reply:
Only if you want code execution to stop. Here I don't want this blocking the popup from going away, cause the api call can take a little while sometimes. Maybe I can look into doing something cleaner with some visual indicator.

Reply:
Oh true I have noticed that, popup stays there for a bit.